.github/workflows/release.yml

```diff
@@ -41,7 +41,26 @@ on:
         default: INFO
         required: false 
         description: "Sets the logging level to get more detailed info."  
-
+      release_bundle_name:
+        description: "Release Bundle Name"
+        required: true
```

Comment:
This can't be `true`. If I choose `create_release_bundle` to be `false`, how would that work?

Reply:
Correction done.